docs/how-to-guides/data-basics/1-setup.md

```diff
@@ -0,0 +1,151 @@
+## Setup
+
+We will build the application as a WordPress plugin, which means you need to have the WordPress itself installed. One way to do it is by following the instructions on [Getting Started](/docs/contributors/code/getting-started-with-code-contribution.md) page. Once your setup is finished, you can follow along with the rest of this tutorial.
+
+## Creating a plugin
+
+We'll do all the development inside of a WordPress plugin. Let's start by creating a `wp-content/plugins/my-first-gutenberg-app` directory in your local WordPress environment. We will need to create three files inside that directory:
+
+* my-first-gutenberg-app.php – to create a new admin page
+* script.js – for our JavaScript application
```

Comment:
```suggestion
* app.js – for our JavaScript application
```

Based on filename usage in the tutorial.

Reply:
Great catch!